server/views/pages/disclaimer.njk

```diff
@@ -26,29 +26,22 @@
           {{ govukCheckboxes({
             idPrefix: 'disclaimer',
             name: 'disclaimer',
-            fieldset: {
-              legend: {
-                text: ' ',
-                isPageHeading: false,
-                classes: "govuk-label"
-              }
-            },
             items: [
               {
                 value: 'disclaimer',
                 text: 'I confirm that I understand'
               }
             ],
-            errorMessage: errors | findError('disclaimer')
+            errorMessage: validationErrors | findError('disclaimer')
```

Comment:
might be good to go back to the original `errors`

Reply:
yep - I was thinking that too